src/App.js

```diff
@@ -1,16 +1,89 @@
 import React from 'react';
 
+import TodoList from './components/TodoComponents/TodoList';
+import TodoForm from './components/TodoComponents/TodoForm';
+import './components/TodoComponents/Todo.css';
+
+const todos = [
+  {
+    task: 'Organize Garage',
+    id: 1528817077286,
+    completed: false
+  },
+  {
+    task: 'Bake Cookies',
+    id: 1528817084358,
+    completed: false
+  }
+];
+
 class App extends React.Component {
-  // you will need a place to store your state in this component.
-  // design `App` to be the parent component of your application.
-  // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor(props) {
+    super(props);
+    this.state = {
+      todos, // same as todos: todos,
+      newTodoText: ''
+    };
+  }
+
+  handleChanges = ev => {
+    this.setState({ [ev.target.name]: ev.target.value });
+  };
+
+  addTodo = e => {
+    e.preventDefault();
+    this.setState({
+      todos: [
+        ...this.state.todos,
+        {
+          task: this.state.newTodoText,
+          id: Date.now(),
+          completed: false
+        }
+      ],
+      newTodoText: ''
+    });
+  };
+
+  toggleDone = (id) => {
+    this.setState({
+      todos: this.state.todos.map(todo => {
+        if (todo.id !== id) {
+          return todo;
+        } else {
+          return {
+            ...todo,
+            completed: !todo.completed
+          };
+        }
+      })
+    });
+  };
+
+  clearDone = e => {
+    e.preventDefault();
+    this.setState({
+      todos: this.state.todos.filter(todo => !todo.completed)
+    });
+  };
+
   render() {
     return (
       <div>
-        <h2>Welcome to your Todo App!</h2>
+        <header>
+          <img src="https://cdn.xl.thumbs.canstockphoto.com/busy-businesswoman-eps-vectors_csp22928432.jpg" alt="busy woman"></img>
```

Comment:
I'm not sure why, but I love the inclusion of this. It gives your app flavor and charm :1st_place_medal: